Tempura/Core/ViewController+Containment.swift

```diff
@@ -32,6 +32,7 @@ extension ViewController {
 /// This is the View that will contain the View of the managed ViewController
 public class ContainerView: UIView {
   
+  /// Layout of the subviews
```

Comment:
Being an overridden method you just have to fallback to the Apple doc with something like 
`- see layoutSubviews()`

Reply:
I like it.